src/Razor/test/testassets/nuget.config

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+  	<clear />
```

Comment:
This means the feeds from our root NuGet.config file aren't visible to the test asset projects. Entirely expected for the build to fail w/o those feeds